Test Report - 2025-04-04_16-36-27
==================================================

Total Tests Run: 2
Successful: 0
Failed: 0
Errors: 2
Run Time: 0:00:00.000253

Errors:
--------------------
test_app (unittest.loader._FailedTest.test_app)
ImportError: Failed to import test module: test_app
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/loader.py", line 396, in _find_test_path
    module = self._get_module_from_name(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/loader.py", line 339, in _get_module_from_name
    __import__(name)
  File "/Users/ganeshbollem/Documents/image-resizer-app/tests/test_app.py", line 5, in <module>
    from app import create_app
  File "/Users/ganeshbollem/Documents/image-resizer-app/app/__init__.py", line 5, in <module>
    from app.config import Config
  File "/Users/ganeshbollem/Documents/image-resizer-app/app/config.py", line 25, in <module>
    os.makedirs(UPLOAD_FOLDER, exist_ok=True)
                ^^^^^^^^^^^^^
NameError: name 'UPLOAD_FOLDER' is not defined



test_pdf (unittest.loader._FailedTest.test_pdf)
ImportError: Failed to import test module: test_pdf
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/loader.py", line 396, in _find_test_path
    module = self._get_module_from_name(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/loader.py", line 339, in _get_module_from_name
    __import__(name)
  File "/Users/ganeshbollem/Documents/image-resizer-app/tests/test_pdf.py", line 6, in <module>
    from app import create_app
  File "/Users/ganeshbollem/Documents/image-resizer-app/app/__init__.py", line 5, in <module>
    from app.config import Config
  File "/Users/ganeshbollem/Documents/image-resizer-app/app/config.py", line 25, in <module>
    os.makedirs(UPLOAD_FOLDER, exist_ok=True)
                ^^^^^^^^^^^^^
NameError: name 'UPLOAD_FOLDER' is not defined



